src/module.js

```diff
@@ -306,11 +357,11 @@ function calculateHalsteadMetrics (data) {
 function nilHalsteadMetrics (data) {
```

Comment:
This function can be inlined since I believe it is only ever called once and only sets properties on the argument.